editor/components/block-list/block-draggable.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Draggable } from '@wordpress/components';
+
+function BlockDraggable( { rootUID, index, uid, layout, isDragging, ...props } ) {
+	const blockDragInsetClassName = classnames( 'editor-block-list__block-draggable', {
+		'is-visible': isDragging,
+	} );
+
+	const transferData = {
+		type: 'block',
+		rootUID: rootUID,
+		fromIndex: index,
+		uid: uid,
+		layout,
+	};
+
+	return (
+		<Draggable className={ blockDragInsetClassName } transferData={ transferData } { ...props }>
+			<div className="inner" ></div>
```

Comment:
The class name does not adhere to standards, and is extremely susceptible to conflicts.

Reply:
If we're expecting extending `Draggable`s to render the child, should this be offered as a component by the `Draggable` ? `Draggable.Inner` ?

Reply:
Space before opening `>` should be removed.

Reply:
Not certain I understand the purpose of a `Draggable.Inner` component.

Reply:
What's the purpose of `<div className="editor-block-list__block-draggable-inner"></div>` ?